reframe/core/pipeline.py

```diff
@@ -82,6 +82,19 @@
 DEPEND_FULLY = 3
 
 
+# Valid systems/environments mini-language
+_N = r'(\w[-.\w]*)'         # name
```

Comment:
Do we want to allow `.` in the names?

Reply:
Yes, because as a future extension I had in mind to allow something like `%proc.arch=broadwell` which it would query the processor object of the partition.